app/src/main/java/br/org/cesar/madmovies/movies/domain/model/Movie.kt

```diff
@@ -8,6 +8,6 @@ import kotlinx.android.parcel.Parcelize
 @Parcelize
 data class Movie(val id: Int, val title: String, val year: String) : Parcelable {
     constructor(movie: RemoteMovie) : this(
```

Comment:
A builder function (`fun Movie(movie: RemoteMovie)`) or a mapper with receiver (`fun RemoteMovie.asMovie()`) is more idiomatic in Kotlin.